pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "conversion_utils"
+version = "0.1.0"
+description = ""
+authors = ["ai-validation-team <ai-validation@minibzk.nl>"]
+readme = "README.md"
+
+[tool.poetry.scripts]
+conversion_utils = "conversion_utils.main:main"
+
+[tool.poetry.dependencies]
+python = "^3.11"
+pandas = "^2.2.2"
+openpyxl = "^3.1.2"
+requests = "^2.32.2"
+
+[tool.poetry.group.test.dependencies]
+pytest = "^8.2.0"
+coverage = "^7.5.1"
+
+[tool.poetry.group.dev.dependencies]
+ruff = "^0.4.4"
+pre-commit = "^3.7.1"
+pyright = "^1.1.362"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+# Ruff settings: https://docs.astral.sh/ruff/configuration/
+[tool.ruff]
+line-length = 120
+target-version = "py311"
+
+[tool.ruff.lint]
+select = ["I", "SIM", "B", "UP", "F", "E"]
+
+fixable = ["ALL"]
+
+[tool.pyright]
```

Comment:
when running pyright ```poetry run pyright``` it gives 22 errors. 

Either remove pyright or make it run without errors.

Reply:
Should work without errors now